### ISSUE TITLE
Printing - Gibberish in many header fields when printing to PDF and real-printing too - Windows

### ISSUE THREAD

**Issue by GreatInca** [richardwilkes/gcs#365]
PDFCreator, Microsoft Print to PDF, PDF Xchange Editor, and real printing. Same fields are gibberish and same gibberish in all.

[Immortal Psychics - David and Michael Yuwono - Microsoft Print to PDF.pdf](https://github.com/richardwilkes/gcs/files/6846217/Immortal.Psychics.-.David.and.Michael.Yuwono.-.Microsoft.Print.to.PDF.pdf)

[Immortal Psychics - David and Michael Yuwono.pdf](https://github.com/richardwilkes/gcs/files/6846219/Immortal.Psychics.-.David.and.Michael.Yuwono.pdf)

zip file has Tobias of Borderlands and David/Michael of Immortal Psychics
[GCS and PDF.zip](https://github.com/richardwilkes/gcs/files/6846250/GCS.and.PDF.zip)

It is doing it to other sheets too.

![GCS_PrintToPDF](https://user-images.githubusercontent.com/73035795/126278797-a3b5271e-80ed-4208-a8e2-178664496306.png)

**Comment by richardwilkes** [comment]
It doesn't do this for me, suggesting there is some sort of corruption on your machine. I'll try this on a few other machines when I get a chance, however, just in case it shows itself on one of the others.
[Untitled.pdf](https://github.com/richardwilkes/gcs/files/6849332/Untitled.pdf)

**Comment by richardwilkes** [comment]
If you are using an alternate font, try setting it back to the defaults and try again to see if that helps.

**Comment by GreatInca** [comment]
I installed on another machine and it is working there.

Looks like Roboto Black and Roboto Medium were not accessible to GCS. Seemed to be rendering, but GCS wasn't finding them in the font settings.   They were shortcut-proxy-installed and the fonts actually lived in the Dell Support Assistant folder.  I nuked them all and their shortcuts and installed a copy of the roboto font family fresh from google and the standard Windows\Fonts folder, and the PDFs are rendering fine now.
